src/elements/content-sidebar/activity-feed/annotation-thread/useAnnotationAPI.js

```diff
@@ -79,7 +113,17 @@ const useAnnotationAPI = ({ annotationId, api, fileId, filePermissions, errorCal
         [errorCallback],
     );
 
-    const handleFetch = React.useCallback(() => {
+    React.useEffect(() => {
+        const getAnnotationSuccess = (fetchedAnnotation: Annotation) => {
+            const { replies: fetchedReplies, ...normalizedAnnotation } = fetchedAnnotation;
+            setAnnotation({ ...normalizedAnnotation });
+            if (fetchedReplies) {
```

Comment:
I think we'll always want to call `setReplies`, otherwise we could get stale replies data.